src/components/Footer/styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+@supports (font-variation-settings: normal) {
+  html { font-family: 'Inter var', sans-serif; 
+    background: #F2F2F2;}
+}
+
+footer {
+    background: #FFFFFF;
+}
```

Comment:
We like to avoid selecting tags, we instead add a class called "footer" to the element and select the class.

It makes the class reusable and avoids tag selector. Take look at the answers in this question to a more depth explanation.